README.md

```diff
@@ -0,0 +1,66 @@
+# `react-native-shake-to-report`
+
+<img src="src/assets/screenshot.png" width="852" />
+
+Using this library, you can add to your app the shake event listener,
```

Comment:
I would change this text to be less technical and more about the problem that the library solves. At this point, it is not relevant how the report is sent (via email or custom), the important bit is that it allows the user to report an issue by shaking the phone.